docs/features/watchdog.md

```diff
@@ -124,26 +136,207 @@ watchdog.on( 'change:state' ( evt, name, currentState, prevState ) => {
+await watchdog.create( {
+	plugins: [
+	    // ...
+	],
+	// ...
+} );
+
+// Add editor instances.
+// You mat also use multiple `ContextWatchdog#add()` calls, each adding a single editor.
+await watchdog.add( [ {
+	id: 'editor1',
+	type: 'editor',
+	sourceElementOrData: document.querySelector( '#editor' ),
+	config: {
+		plugins: [ Essentials, Paragraph, Bold, Italic ],
+		toolbar: [ 'bold', 'italic', 'alignment' ]
+	},
+	creator: ( element, config ) => ClassicEditor.create( element, config )
+}, {
+	id: 'editor2',
+	type: 'editor',
+	sourceElementOrData: document.querySelector( '#editor' ),
+	config: {
+		plugins: [ Essentials, Paragraph, Bold, Italic ],
+		toolbar: [ 'bold', 'italic', 'alignment' ]
+	},
+	creator: ( element, config ) => ClassicEditor.create( element, config )
+} ] );
+
+// Or:
+await watchdog.add( {
+	id: 'editor1',
+	type: 'editor',
+	sourceElementOrData: document.querySelector( '#editor' ),
+	config: {
+		plugins: [ Essentials, Paragraph, Bold, Italic ],
+		toolbar: [ 'bold', 'italic', 'alignment' ]
+	},
+	creator: ( element, config ) => ClassicEditor.create( element, config )
+} );
+
+await watchdog.add( {
+    id: 'editor2',
+    type: 'editor',
+    sourceElementOrData: document.querySelector( '#editor' ),
+    config: {
+        plugins: [ Essentials, Paragraph, Bold, Italic ],
+        toolbar: [ 'bold', 'italic', 'alignment' ]
+    },
+    creator: ( element, config ) => ClassicEditor.create( element, config )
+} );
+```
+
+To destroy one of the item instances use {@link module:watchdog/contextwatchdog~ContextWatchdog#remove `ContextWatchdog#remove`}:
+
+```js
+await watchdog.remove( [ 'editor1', 'editor2' ] );
 
-Both, the {@link module:watchdog/watchdog~Watchdog#constructor `Watchdog#constructor`} and the {@link module:watchdog/watchdog~Watchdog.for `Watchdog.for`} methods accept a {{@link module:watchdog/watchdog~WatchdogConfig configuration object} with the following optional properties:
+// Or:
+await watchdog.remove( 'editor1' );
+await watchdog.remove( 'editor2' );
+```
 
-* `crashNumberLimit` - A threshold specifying the number of editor errors (defaults to `3`). After this limit is reached and the time between last errors is shorter than `minimumNonErrorTimePeriod` the watchdog changes its state to `crashedPermanently` and it stops restarting the editor. This prevents an infinite restart loop.
-* `minimumNonErrorTimePeriod` - An average amount of milliseconds between last editor errors (defaults to 5000). When the period of time between errors is lower than that and the `crashNumberLimit` is also reached the watchdog changes its state to `crashedPermanently` and it stops restarting the editor. This prevents an infinite restart loop.
+<info-box>
+	Examples presents the "synchronous way" of the integration with the context watchdog feature, however it's not needed to wait for the promises returned by the `create()`, `add()` and `remove()` methods. There might be a need
```

Comment:
I don't understand this info box, to be honest. I don't know what you wanted to convey here and whether it is actually important.

Reply:
I wanted to say that the promises are chained anyway so there's no need to wait for these promises as some integrations may require running multiple `add()` and `remove()` functions in the watchdog at the same time.